pandas-stubs/core/series.pyi

```diff
@@ -121,7 +121,41 @@ class Series(Generic[_DType]):
         self, other: Series, method: Literal["pearson", "kendall", "spearman"] = ...
     ) -> float: ...
     def count(self) -> int: ...
-    def fillna(self, value: Union[_DType, Dict, Series, DataFrame]) -> Series: ...
+    @overload
+    def drop(
+        self, labels: Union[_str, List[_str], Index], axis: _AxisType = ..., inplace: bool = ...
+    ) -> Series[_DType]: ...
+    @overload
+    def drop(self, *, index: Union[List[_str], Index]) -> Series[_DType]: ...
+    @overload
+    def drop(self, *, columns: Union[_str, List[_str], Index]) -> Series[_DType]: ...
+    def drop_duplicates(self, keep: Union[_str, bool] = ...) -> Series[_DType]: ...
+    @overload
+    def duplicated(self, keep: Literal["first"] = ...) -> Series[_DType]: ...
+    @overload
+    def duplicated(self, keep: Literal["last"] = ...) -> Series[_DType]: ...
+    @overload
+    def duplicated(self, keep: Literal[False] = ...) -> Series[_DType]: ...
+    @overload
+    def fillna(
+        self,
+        value: Union[_DType, Dict, Series, DataFrame],
+        method: _str = ...,
+        axis: Optional[_AxisType] = ...,
+        inplace: Literal[True] = ...,
```

Comment:
If `inplace` is `True`, doesn't this function return `None`?

Reply:
Yes, good catch!